src/components/admin/form-components/validated-fields.jsx

```diff
@@ -32,3 +40,5 @@ export const ShortRequiredTextField = withValidate(
   cannotNull,
   shorterThanDbVarChar,
 )(DirectTextField);
+
+export const SlugField = withValidate(isPrettySlug)(ShortRequiredTextField);
```

Comment:
Does withValidate work when composed with itself like this? (since ShortRequiredTextField already has a withValidate call)

Reply:
Yup, the beauty of composition.